BPStatusBarAlert/Classes/BPStatusBarAlert.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import UIKit
```

Comment:
if using `import Foundation`, `import Foundation` do not need anymore.
I'd like to remove it :)